crates/proto/src/xfer/dnssec_dns_handle.rs

```diff
@@ -688,17 +705,6 @@ where
         })
         .collect::<Vec<_>>();
 
-    // if there are no available verifications, then we are in a failed state.
-    if verifications.is_empty() {
```

Comment:
I think we should keep this, unless we can describe how this case is otherwise is covered. I can't remember where else we are setting `Bogus` when there are no RRSIGs. The earlier return on line 288 is for when there are no records to verify, i.e. empty, this is for the case where there were records, but we got no verifications... this could be due to no RRSIGs available. I think they are different cases.